docs/reference/generated/accordion-root.json

```diff
@@ -35,12 +35,12 @@
       "description": "Whether to loop keyboard focus back to the first item\nwhen the end of the list is reached while using the arrow keys."
     },
     "orientation": {
-      "type": "'horizontal' | 'vertical'",
+      "type": "\"horizontal\" | \"vertical\"",
```

Comment:
Nitpick, could we use consistently single or double quotes for the type and the default value? The default value uses single quite, while the type is now double quote, it looks a bit off in the docs: https://deploy-preview-1537--base-ui.netlify.app/react/components/accordion#root